src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/manual-remove/ManualRemove.test.tsx

```diff
@@ -8,6 +8,14 @@ import { composeStory } from "@storybook/react";
 import { axe } from "jest-axe";
 import { userEvent } from "@testing-library/user-event";
 
+jest.mock("next/navigation", () => {
+  return {
+    useRouter: () => ({
+      refresh: jest.fn(),
+    }),
+    usePathname: jest.fn(),
+  };
+});
```

Comment:
Is this mock used in this file?

Reply:
It is, in the sense that it prevents the original code from running, because that is trying to access browser APIs that don't exist in Node. But I'm not making assertions on it no - though I suppose I should? f83b73526